xmodule/capa/safe_exec/safe_exec.py

```diff
@@ -178,6 +179,21 @@ def safe_exec(
         else:
             emsg = None
 
+        # Run the code in both the remote codejail service as well as the local codejail
+        # when in darklaunch mode.
+        if is_codejail_in_darklaunch():
+            data = {
```

Comment:
I think this conditional code needs to be in a try/except that may call `record_exception` and log any exception, but then swallows it and moves on, so the darklaunch can't cause failures if there are any unknown issues with the codejail service.